Frameworks/include/UIPopoverPresentationControllerInternal.h

```diff
@@ -0,0 +1,32 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#ifndef _UIPOPOVERPRESENTATIONCONTROLLERINTERNAL_H_
+#define _UIPOPOVERPRESENTATIONCONTROLLERINTERNAL_H_
+
+@interface UIPopoverPresentationController () {
+@public
```

Comment:
Even the #pragma once isn't strictly necessary since we're #import ing these - but added them to all headers.